library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
This file is only there at that place because I was too dumb to place it elsewhere when I switched the parser in the past. (It was an issue with the autoloader)